test/interpose/CMakeLists.txt

```diff
@@ -16,6 +16,10 @@ set(FUNS_pthread_create pthread_create pthread_join)
 set(INCL_pthread_mutex ${INCL_pthread_create})
 set(FUNS_pthread_mutex pthread_mutex_lock pthread_mutex_unlock
                        pthread_mutex_trylock)
+
+if(HAVE_PTHREAD_CLOCKWAIT_LOCK)
```

Comment:
Where do you define this? in the C file?

Reply:
This is a CMakeLists.txt are you sure this is working? Is the then case ever being triggered?